src/main/scala/com/redislabs/provider/redis/redisFunctions.scala

```diff
@@ -390,6 +402,28 @@ object RedisContext extends Serializable {
     conn.close()
   }
 
+  /**
+    * @param arr hashName/k/vs which should be saved in the target host
+    *            save all the k/vs to hashName(list type) to the target host
```

Comment:
@charsyam I'm not understanding the second line of the comment. What do you mean by `hashName(list type)`?

Reply:
Thanks @simalince, It is just my mistake. I will remove. :) Thanks